manifests/quick-start/base/httpbin/my-httpbin-cred-secret.yaml

```diff
@@ -12,7 +12,7 @@ stringData:
   # for oauth2 authentication
   clientID: admin
   clientSecret: password
-  tokenURL: "http://httpbin:9100/response-headers?access_token=faketoken&token_type=Bearer" # this URL will return a body with the "access_token" field set and can simulate an oauth token flow
+  tokenURL: "http://httpbin:9100/base64/YWNjZXNzX3Rva2VuPWZha2V0b2tlbg==" # this URL will return a body with the "access_token" field set and can simulate an oauth token flow
```

Comment:
Had to change this because go-httpbin gives different responses for this endpoint: https://github.com/mccutchen/go-httpbin/issues/141